components/layout_2020/table/layout.rs

```diff
@@ -1044,7 +1044,7 @@ impl<'a> TableLayout<'a> {
             self.column_measures[*column_index]
                 .content_sizes
                 .max_content !=
-                Au(0)
+                Au::zero()
```

Comment:
Ditto

Reply:
This works here because style::Zero is used in this module, but let me know what I should do about the other case.